cmd/iq.go

```diff
@@ -42,7 +42,8 @@ func init() {
 	rootCmd.AddCommand(iqCmd)
 
 	pf := iqCmd.PersistentFlags()
-	pf.StringVar(&operating, "os", "", "Specify a value for the operating system type you want to scan (alpine, debian, fedora). Useful if autodetection fails and/or you want to explicitly set it.")
+	pf.StringVar(&packageManager, "os", "", "Specify a value for the operating system type you want to scan (alpine, debian, fedora). Useful if autodetection fails and/or you want to explicitly set it.")
```

Comment:
So dumb question if we do this and set it to `&packageManager` wont that break everyone that is in theory currently passing in a real value here?? Or does Cobra just stop you since its been deprecated and spits out a message??

Reply:
good question -- no it just throws the deprecation warning and continues.

i thought i'd tested this successfully, but now i'm not sure how we could get desired behavior without some more logic to map the os names to package manager names... i'm in the middle of distro hopping (perfect timing :-) ) so need to get docker going again and some VMs spun up to test this adequately. i'll hammer that out this weekend.

thanks for the feedback.